src/api/app/models/user.rb

```diff
@@ -952,6 +952,7 @@ def hash_string(value)
 #
 #  id                            :integer          not null, primary key
 #  adminnote                     :text(65535)
+#  bio                           :text(65535)
```

Comment:
A short biography is typically four sentences, approximately 150 to 200 words or less in length.

Reply:
That's true, it makes no sense to store a text for this. I've just changed to String, 255 characters, what I think is not a bad number for a bio. But we can play afterwards with validations in case we decide to allow less characters for design decitions.